src/test/java/org/graylog/integrations/notifications/types/SlackEventNotificationTest.java

```diff
@@ -0,0 +1,167 @@
+package org.graylog.integrations.notifications.types;
+
+import com.floreysoft.jmte.Engine;
+import com.google.common.collect.ImmutableList;
+import org.graylog.events.notifications.*;
```

Comment:
Prefer specific imports to `*`

Reply:
I have adjusted my intellij settings to apply the preferences.:)